crates/biome_cli/src/service/mod.rs

```diff
@@ -72,11 +72,11 @@ type JsonRpcResult = Result<Box<RawValue>, TransportError>;
 /// This structs holds an instance of the `tokio` runtime, as well as the
 /// following fields:
 /// - `write_send` is a sender handle to the "write channel", an MPSC channel
-/// that's used to queue up requests to be sent to the server (for simplicity
-/// the requests are pushed to the channel as serialized byte buffers)
+///     that's used to queue up requests to be sent to the server (for simplicity
```

Comment:
Just wondering: does the rule require 4 spaces? Or 2 spaces could be enough?

Reply:
Two spaces. I used the tab to be fast